app/routes/view-application.js

```diff
@@ -28,8 +29,10 @@ module.exports = {
 
       const status = upperFirstLetter(application.status.status.toLowerCase())
       const statusClass = getStyleClassByStatus(application.status.status)
+      const mappedAuth = mapAuth(request)
       const withdrawLinkStatus = ['IN CHECK', 'AGREED']
```

Comment:
minor - could these statuses be abstracted to single place as constants/enums and is it possible to use statusId instead of the string value?